src/OrchardCore.Modules/OrchardCore.Lists/Feeds/ListPartFeedHandler.cs

```diff
@@ -11,8 +11,10 @@ public override Task GetContentItemAspectAsync(ContentItemAspectContext context,
         {
             return context.ForAsync<FeedMetadata>(feedMetadata =>
             {
-                // If the value is not defined, it will be represented as null
+                // Disable the feed if the value is not defined
+                feedMetadata.EnableFeedProxyUrl = part.Content.EnableFeedProxyUrl ?? false;
```

Comment:
I think this might influence existing feeds when updating from older version?
They will have null value in this new prop, so it could disable them all? I think it should be true in default

Reply:
Good point, let me think about